elementary/monitor/data_monitoring/alerts/integrations/slack/slack.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         self.tracking = tracking
         self.override_config_defaults = override_config_defaults
         self.message_builder = SlackAlertMessageBuilder()
-        super().__init__()
+        super().__init__(*args, **kwargs)
```

Comment:
do you sometimes pass on unknown arguments here? because if not I'd remove this entirely (from the init arguments as well)